### ISSUE TITLE
Vuex: `$store` undefined

### ISSUE THREAD

**Issue by symonxdd** [Kocal/vue-web-extension#678]
**Describe the bug**
The global Vuex object: `$store`, appears to be `undefined`. See screenshot below.

**To Reproduce**
1. Follow setup instructions from this repo's README
    a. `vue create --preset kocal/vue-web-extension my-extension`
    b. `cd my-extension`
    c. `npm run serve`
2. Define sample variable and method in the `index.js` in **src/store/**: 

```js
import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

export default new Vuex.Store({
    state: {
        apples: []
    },
    mutations: {
        _updateApples(state, payload) {
            state.apples= payload;
        },
    },
    actions: {
        getUserSettings: async function ({ commit, state }, payload) {
            chrome.storage.local.get({
                apples: []
            }, items => {
                commit("_updateApples", items.apples);
            });
        },
    },
    modules: {
    }
})
```

`App.vue` in **src/popup**:
```js
<template>
	<div>
		<h2 v-if="$store">Vuex works!</h2>
		<h2 v-else>I don't think so</h2>
	</div>
</template>

<script>
export default {
	name: "App",
	mounted() {
		this.setSampleUserSettings();
	},
	methods: {
		setSampleUserSettings: function () {
			this.$store.dispatch("setSampleUserSettings");
		},
	},
};
</script>
```

**Expected behavior**
The global Vuex variable: `$store` to not be undefined.

**Screenshots**  

Chrome Console errors:  
![image](https://user-images.githubusercontent.com/45405238/152905263-b9ce782d-5ea7-4bfd-af86-a2c3d51f44d1.png)

**Desktop (please complete the following information):**
- [vue-web-extension](https://github.com/Kocal/vue-web-extension) version: v2.0.0
 - OS: Windows 11
 - Node.js version: v14.15.0

**Comment by jjone36** [comment]
Can anyone please answer this issue? 
I'm also having the same kind of a problem he's having. 
It just seems not able to read store at all.. (state, commit, dispatch.. all gives the undefined message)

**Comment by SymonBlazejczakPXL** [comment]
> Can anyone please answer this issue? I'm also having the same kind of a problem he's having. It just seems not able to read store at all.. (state, commit, dispatch.. all gives the undefined message)

**TL;DR**

Add the import statement for the **store** inside **src/popup/main.js** like so:

`import store from '../store'`

and attach it to the Vue object

```javascript
new Vue({
    store,
    el: '#app',
    render: h => h(App)
})
```

It would've been nice if the developer of this CLI preset would document how the overall folder-structure works. It appears that there's no single `main.js` file we're used to with a normal Vue app, instead the **popup** & **options** pages have their own `main.js` file which acts as an entry point to both of them.


**My `main.js` inside /popup:**
```javascript
import Vue from 'vue'
import App from './App.vue'
import router from '../router/'
import store from '../store'

/* eslint-disable no-new */
new Vue({
    store,
    router,
    el: '#app',
    render: h => h(App)
})
```

**Comment by jjone36** [comment]
> It would've been nice if the developer of this CLI preset would document how the overall folder-structure works. 

Can't agree more..
Thank you so much for the comment. It finally works 👍🏻👍🏻
